.circleci/config.yml

```diff
@@ -252,8 +252,6 @@ jobs:
       - run:
           name: Tests TypeScript definitions
           command: pnpm typescript:ci
-          environment:
-            NODE_OPTIONS: --max-old-space-size=1536
```

Comment:
I'm not sure if the assumption that this is no longer necessary is correct. 🤔 
We could end up running into OOM issues even with concurrency of `1` if a particular package is super memory intensive. 🤔 
IMHO, I'd leave it as 3.5GB just to be safe. 😃